vmm/src/device_manager.rs

```diff
@@ -1995,6 +1995,10 @@ impl DeviceManager {
                     Endpoint::File(stdout)
                 }
             }
+            ConsoleOutputMode::Socket => {
+                error!("socket option is not supported for Console devices");
+                return Ok(None);
```

Comment:
Should this return an error rather than `Ok` ?

Reply:
Now that I think about this again, it makes sense to return an error instead of continuing here. This will allow user to retry a working console option, instead of expecting him/her to go through the logs to figure out what happened. 

I will update this.